app/client/components/SearchAndFilterPage.tsx

```
@@ -0,0 +1,342 @@
+    }
+
+    setPriceRange(newRange)
+  }
+
+  const handleInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    index: number
+  ) => {
+    let value = parseInt(e.target.value, 10)
+
+    if (isNaN(value)) return
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)